app/code/Magento/Tax/Plugin/Checkout/CustomerData/Cart.php

```diff
@@ -66,7 +66,7 @@ public function afterGetSectionData(\Magento\Checkout\CustomerData\Cart $subject
             foreach ($result['items'] as $key => $itemAsArray) {
                 if ($item = $this->findItemById($itemAsArray['item_id'], $items)) {
                     $this->itemPriceRenderer->setItem($item);
-                    $this->itemPriceRenderer->setTemplate('checkout/cart/item/price/sidebar.phtml');
+                    $this->itemPriceRenderer->setTemplate('Magento_Tax::checkout/cart/item/price/sidebar.phtml');
```

Comment:
Please, revert fix in places when the template specified dynamically

Reply:
Done.